conservative_crawling_ai/agent.py

```diff
@@ -0,0 +1,106 @@
+import random
+from shared.probabilities import handler
+from shared.ai import agent
+
+
+def normalise(arr):
+    sum_arr = sum(arr)
+    if sum_arr:
+        return [i/sum_arr for i in arr]
+    return arr
+
+
+def elementwise_mul(first_array, second_array):
+    return [a*b for a, b in zip(first_array, second_array)]
+
+def compute_sampling_weights(bet_probs, bet_probs_generic):
```

Comment:
The function should check both arguments are length (>=) 88, or much better, check:
```
if len(bet_probs) != len(bet_probs_generic):
    raise ValueError("Bet probability arrays are not of equal length")
```
and just iterate
```
... for i in range(len(bet_probs))]
```